packages/app/src/specs/banners/RecordBanner.vue

```diff
@@ -16,47 +16,16 @@
     }"
     @update:model-value="value => emit('update:modelValue', value)"
   >
-    <p class="mb-24px">
-      {{ t('specPage.banners.record.content') }}
-    </p>
-
-    <TerminalPrompt
-      :command="recordCommand"
-      :project-folder-name="query.data?.value?.currentProject?.title"
-      class="bg-white max-w-900px"
-    />
+    <RecordPrompt />
   </TrackedBanner>
 </template>
 
 <script setup lang="ts">
-import { gql, useQuery } from '@urql/vue'
 import RecordIcon from '~icons/cy/action-record_x16.svg'
 import { useI18n } from '@cy/i18n'
-import TerminalPrompt from '@cy/components/TerminalPrompt.vue'
 import TrackedBanner from './TrackedBanner.vue'
 import { BannerIds } from '@packages/types'
-import { RecordBannerDocument } from '../../generated/graphql'
-import { computed } from 'vue'
-
-gql`
-query RecordBanner {
-  currentProject {
-    id
-    title
-    currentTestingType
-    cloudProject {
-      __typename
-      ... on CloudProject {
-        id
-        recordKeys {
-          id
-          key
-        }
-      }
-    }
-  }
-}
-`
+import RecordPrompt from '@packages/frontend-shared/src/gql-components/RecordPrompt.vue'
```

Comment:
Should this be in `packages/app`? I don't think we will ever use the `RecordPrompt` in Launchpad.

Reply:
I put it in `frontend-shared` because that's where `Auth` is, which I think is where `RecordRunModal` will be used 🤔 not sure where to put it

Reply:
If I understand the linked issue correctly the intent is to eventually give users a push to record as part of the login flow (login modal => connect modal => record modal), so it likely will need to be a shared component